src/cards/departure-card.card.js

```diff
@@ -0,0 +1,524 @@
+/**
+ * @typedef {CardConfig&{
+ *    layout: string,
+ *    departures_attribute: string,
+ *    departure_properties: {
+ *        time: string,
+ *        delay?: string,
+ *        cancelled?: string,
+ *        train?: string,
+ *        direction: string,
+ *        platform?: string,
+ *        next_stations?: string,
+ *    },
+ *    destination_filter?: string,
```

Comment:
Maybe this should allow an array. This might be relevant when multiple destinations should be allowed. Here a simplified example:

`Train 1: A-----B-----C-----D-----E`

`Train 2: A-----B-----C`

My destination is C but for Train 1, C is not listed in next_stations.